### ISSUE TITLE
Legends not working with JAWS screen reader

### ISSUE THREAD

**Issue by mae-financial-com** [highcharts/highcharts#15766]
#### Expected behaviour
When navigating to legends using keyboard Tab key with JAWS screen reader running the focus goes to legends but they are not read as button as was the case previously(Version 7.2.2 of highcharts). Legends are read as graphic by JAWS. Also they cannot be toggled using Spacebar or Enter key which is usually how button/legends work.

#### Actual behaviour
The legends should be read as buttons and should work with spacebar and enter key to toggle them.

#### Live demo with steps to reproduce
Not able to reproduce on jsfiddle.
Issue raised on highcharts forum with details - [Detailed description](https://www.highcharts.com/forum/viewtopic.php?f=9&t=46517#p166809)

#### Product version
Highcharts 9.0.1

#### Affected browser(s)
All browsers Chrome, Brave, Edge

**Comment by KacperMadej** [comment]
Thank you for reporting the problem.

**Comment by mae-financial-com** [comment]
This can be marked as resolved as issue got resolved after this - https://github.com/highcharts/highcharts/issues/15890
